tests/test_resource_tracker.py

```diff
@@ -157,6 +172,48 @@ def create_resource(rtype):
                                                 errno.ENOENT))
         assert re.search(expected, err) is not None
 
+    def test_resource_tracker_refcounting(self):
+        cmd = '''if 1:
+        import os
+        import tempfile
+        import time
+        from loky.backend import resource_tracker
+
+        resource_tracker.VERBOSE=True
```

Comment:
nitpick

```suggestion
        resource_tracker.VERBOSE = True
```

Reply:
Actually, why is this needed? There does not seem to be any check in the output of the subprocess' stdout.

Reply:
Yes, this may be a copy-paste leftover.

Reply:
While I'm at it, the logging of the `ResourceTracker` is a little bit awkward (uses `sys.stderr.write`). I'm checking if we can instead use some a more structured logging API.

Reply:
The test has been updated to report stderr in case of assertion failure but the original style nitpick still holds :)